ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/Reporter.scala

```diff
@@ -8,7 +8,7 @@ import java.io.{PrintStream, PrintWriter, StringWriter}
 import scala.util.Try
 
 trait Reporter[A] {
-  def report(results: Vector[LedgerTestSummary]): A
+  def report(results: Vector[LedgerTestSummary], suffix: String): A
```

Comment:
Please use a single name to refer to this information. The original `**identifierSuffix` seems to be a sensible choice.

Reply:
fixed